packages/@lwc/wire-service/src/index.ts

```diff
@@ -110,8 +127,8 @@ function isDifferentConfig(
     return params.some((param) => newConfig[param] !== oldConfig[param]);
 }
 
-export class WireAdapter {
```

Comment:
intentionally removing this export. this class is internal only and used as a bridge between the old with protocol (via events) and the new wire protocol.